src/main/java/roomescape/login/LoginController.java

```diff
@@ -0,0 +1,40 @@
+package roomescape.login;
+
+import jakarta.servlet.http.Cookie;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.Map;
+
+@RestController
+public class LoginController {
+
+    private LoginService loginService;
```

Comment:
[제안 🗣️ ]

불변 신경써주는 건 어떨까요?
```suggestion
    private final LoginService loginService;
```

Reply:
헉! 실수했습니다 꼼꼼하게 체크하겠습니다